docs/basics.md

```diff
@@ -98,6 +99,32 @@ Install Connaisseur via Helm:
     helm install connaisseur connaisseur/connaisseur --atomic --create-namespace --namespace connaisseur -f values.yaml
     ```
 
+=== "Kubernetes manifests"
```

Comment:
Should the `### Get the code/chart` section also get a Kubernetes manifests content tab such that you can use the same content tab for everything?

Reply:
thought about it, but don't see what to say there, as it is literally just doing the same as for `git` and in that case we need to rename the "git repo" part to "git repo (helm)" or something and we start changing more and more for minimal added consistency value